toolchain/parser/parser_handle_expression.cpp

```diff
@@ -9,6 +9,8 @@ namespace Carbon {
 CARBON_DIAGNOSTIC(
     OperatorRequiresParentheses, Error,
     "Parentheses are required to disambiguate operator precedence.");
```

Comment:
With this change, is this diagnostic now only used in one place such that it could be relocated there too?